pkg/kubectl/resource/helper.go

```diff
@@ -52,7 +52,7 @@ func NewHelper(client RESTClient, mapping *meta.RESTMapping) *Helper {
 	}
 }
 
-func (m *Helper) Get(namespace, name string, export bool) (runtime.Object, error) {
+func (m *Helper) Get(namespace, name string, export, includeUninitialized bool) (runtime.Object, error) {
```

Comment:
Let's leave this function as is. I don't see any use case to hide an uninitialized object if user specifically asks for it.